openpype/hosts/houdini/api/lib.py

```diff
@@ -460,3 +476,60 @@ def reset_framerange():
     hou.playbar.setFrameRange(frame_start, frame_end)
     hou.playbar.setPlaybackRange(frame_start, frame_end)
     hou.setFrame(frame_start)
+
+
+def load_creator_code_to_asset(
+        otl_file_path, node_type_name, source_file_path):
+    # type: (str, str, str) -> None
+    # Load the Python source code.
+    with open(source_file_path, "rb") as src:
+        source = src.read()
+
+    # Find the asset definition in the otl file.
+    definitions = [definition
+        for definition in hou.hda.definitionsInFile(otl_file_path)
+        if definition.nodeTypeName() == node_type_name]
+    assert(len(definitions) == 1)
+    definition = definitions[0]
+
+    # Store the source code into the PythonCook section of the asset.
+    definition.addSection("PythonCook", source)
+
+
+def get_main_window():
+    """Acquire Houdini's main window"""
+    if self._parent is None:
+        self._parent = hou.ui.mainQtWindow()
+    return self._parent
+
+
+def parm_to_template(key, value):
```

Comment:
Function name feels a bit confusing - it's not actually turning a `hou.Parm` into a template. It's take a parameter name and a default value and for that define a corresponding parm template.

Options/Ideas:
- `parm_template_from_value(name, value)`
- `auto_parm_template(name, value)`